cmd/extensions.rst

```diff
@@ -0,0 +1,14 @@
+Extensions
```

Comment:
we cannot have this file without modifying Makefile. The default build target expects everything under `cmd/` is a direcrory.

Do we need  `extensions.rst` here?

Reply:
Moved to /docs for now.